DashKitUI/View Controllers/DashUICoordinator.swift

```diff
@@ -142,27 +142,36 @@ class DashUICoordinator: UINavigationController, PumpManagerCreateNotifying, Pum
 
             let viewModel = PairPodViewModel(podPairer: pumpManager, navigator: self)
 
-            viewModel.didFinish = { [weak self] in
-                self?.stepFinished()
-            }
-            viewModel.didCancel = { [weak self] in
-                self?.setupCanceled()
-            }
+            viewModel.didFinish = stepFinished
+            viewModel.didCancelSetup = setupCanceled
+            viewModel.didRequestDeactivation = { self.navigateTo(.deactivate) }
+            
             let view = hostingController(rootView: PairPodView(viewModel: viewModel))
             view.navigationItem.title = LocalizedString("Pod Pairing", comment: "Title for pod pairing screen")
             return view
+        case .confirmAttachment:
+            guard let pumpManager = pumpManager else {
+                fatalError("Need pump manager for confirm attachment screen")
+            }
+            let view = AttachPodView {
```

Comment:
[nit] This form is confusing to me. I had to go look at `AttachPodView` to figure out what the first closure was used for. Perhaps specifically call out each, `didConfirmAttachment` and `didRequestDeactivation`?